index.html

```diff
@@ -6,11 +6,83 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="./style.css">
 </head>
 
+<!-- Header space -->
+
 <body>
-    <!-- Add your markup here -->
+    
+    <header>
+
+<section class="bakery-icon">
+    <div class="cupcake-icon">
+        <img src="./design/cupcake_logo.png" alt="Bakery allusive icon">
+    </div>
+<br>
+<div>
+    <p class="welcome-msg">The best cakes in town delivered to your door</p>
+</div>
+
+</section>
+
+<!-- Menu / Navigation bar-->
+
+<nav>
+    <section class="mobile-nav">
+
+
+        <img src="./design/menu-hamburger.svg" class="hamburger-icon" onclick="navMenu()">
+          <div class="menu-nav" id="links">
+            
+            <ul>
+                <li><a href="#HOME">HOME</a></li>
+                <li><a href="#CAKES">CAKES</a></li>
+                <li><a href="#ORDERING">ORDERING</a></li>
+                <li><a href="#LESSONS">LESSONS</a></li>
+                 <li><a href="#ABOUT">ABOUT</a></li>
+            </ul>
+        </div>
+    </section>
+</nav>
+
+</header>
+
+<!-- Main section-->
+    <main>
+        <section class="welcome"> 
+            <div class="welcome-content">
+                <h1>WELCOME!</h1>
+                <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."</p>
+            </div>  
+            <div class="welcome-img">
+                <img class="welcome-image" src="./design/bday_cake.jpg" alt="Cake picture">
+            </div>
+        </section>
+        <section class="products">
+                <img src="./design/blue_cupcake.jpg" alt="cupcake w/ blue background">
+                <img src="./design/pink_macaroon.jpg" alt="macaroons w/ pink background">
+                <img src="./design/yellow_cookie.jpg" alt="cookie w/ yellow background">
+                <img src="./design/blue_donut.jpg" alt="donut w/ blue background">
+        </section>
+    </main>
 </body>
 
+<!-- Footer / Social media / Contact -->
+
+<footer>
+    <section class="social-media">
+        <p>Find us</p>
+    </section>
```

Comment:
This doesn't need a separate section - it's just the text above the media-icons, so it can probably be just the `p` by itself